src/Aspire.Dashboard/Configuration/DashboardOptions.cs

```diff
@@ -73,10 +74,18 @@ public sealed class OtlpOptions
     public OtlpAuthMode? AuthMode { get; set; }
     public string? EndpointUrl { get; set; }
```

Comment:
Can we rename this to `GrpcEndpointUrl` for consistency?

Reply:
done